ibm/service/vpc/resource_ibm_is_lb.go

```diff
@@ -469,6 +476,9 @@ func lbGet(d *schema.ResourceData, meta interface{}, id string) error {
 
 	d.Set(isLBResourceGroup, *lb.ResourceGroup.ID)
 	d.Set(isLBHostName, *lb.Hostname)
+	if lb.UDPSupported != nil {
```

Comment:
isLBUdpSupported is supported only for network load balancer hence i feel there should also be optional: true.
Please reverify 
https://pages.github.ibm.com/vpc/vpc-spec-artifacts/branch/master/public/development-tomorrow.html#/Load%20balancers/create_load_balancer